src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml.cs

```diff
@@ -23,10 +23,8 @@
 using System.Windows.Controls;
 using System.Windows.Controls.Primitives;
 using System.Windows.Data;
-using System.Windows.Documents;
```

Comment:
The 2 using blocks were were unused before this change--this just cleans them up